docs/rest-api.yml

```diff
@@ -387,13 +387,13 @@ paths:
         type: string
       responses:
         '200':
-          description: The user identified by the API token
+          description: The user is identified by the API token
```

Comment:
The content of the response here is the user identified by the token. Adding 'is' no longer describes the content of the response.